packages/components/src/Toast/showToast.tsx

```diff
@@ -7,26 +7,27 @@ import React, {
   useRef,
   useState,
 } from "react";
-import { render } from "react-dom";
+// According to react, it's imported within the package
+// https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis
+// eslint-disable-next-line import/no-internal-modules
+import { createRoot } from "react-dom/client";
 import { Toast, ToastProps, ToastRef } from "./Toast";
 import styles from "./Toast.css";
 
-export function showToast(props: ToastProps) {
-  createDocumentToast(props);
-}
+const targetId = "atlantis-toast-element";
+let target = document.querySelector(`#${targetId}`);
 
-function createDocumentToast(props: ToastProps) {
-  const targetId = "atlantis-toast-element";
-  let target = document.querySelector(`#${targetId}`);
+if (!target) {
+  target = document.createElement("div");
+  target.id = targetId;
+  target.classList.add(styles.wrapper);
+  document.body.appendChild(target);
+}
 
-  if (!target) {
-    target = document.createElement("div");
-    target.id = targetId;
-    target.classList.add(styles.wrapper);
-    document.body.appendChild(target);
-  }
+const root = createRoot(target);
 
-  render(<ToasterOven {...props} />, target);
+export function showToast(props: ToastProps) {
+  root.render(<ToasterOven {...props} />);
```

Comment:
As discussed in Slack, we need this change to make the tests pass in JO.
```suggestion
  export function showToast(props: ToastProps) {
    if (document?.body?.contains(target)) {
      root.render(<ToasterOven {...props} />);
    }
}
```